src/pages/index.js

```diff
@@ -3,6 +3,7 @@ import Slider from "react-input-slider";
 import fetch from "isomorphic-unfetch";
 import Error from "../Components/Error";
 import DrumPad from "../Components/DrumPad";
+import DrumLoopTile from "../Components/DrumLoopTile/index";
```

Comment:
- [ ] For now until we have our designs we are not going have the DrumLoopTile appear on index.js. Can you remove it from here for now?

Reply:
okay


> Hi @talk2coded,
> 
> Awesome work!! :) Thanks for the contribution. I just ask that you fix the merge conflict.
> 
> Also I strongly encourage you to write a test for this component. To start you can write a test that will just assert that this component renders properly with default props. If you need help with it please let me know :)

how do i do that ?

Reply:
Great! Please check the other components (in `src/components` folder) and look for files called `index.test.js` for examples on how to do this. Have you used `jest` and/or `enzyme` before? If not please refer to this [resource](https://www.freecodecamp.org/news/testing-react-hooks/) as a guide

Please try to do this and let me know how everything goes.

Reply:
i have done it